Lib/Google/GoogleAuthContainer.php

```diff
@@ -16,4 +16,22 @@ class GoogleAuthContainer extends AuthContainer {
 	 */
 	public $service;
 
+
+	/** @var  GoogleAuthManager */
+	private $_manager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "$_manager" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)